traja/tests/test_models.py

```diff
@@ -59,7 +59,7 @@ def test_aevae_jaguar():
     trainer = HybridTrainer(model=model, optimizer_type="Adam", loss_type="huber")
 
     # Train the model
-    trainer.fit(data_loaders, model_save_path, epochs=10, training_mode="forecasting")
+    trainer.fit(data_loaders, model_save_path, epochs=10, training_mode="forecasting", validate_every=5, test_every=10)
```

Comment:
Could we test `validate_every=None` and `test_every=None` as well, please?

Reply:
Done.